CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+- Updated the “Send” button in “Send to your Devices” window to be displayed in the header section. ([#8329](https://github.com/brave/brave-ios/issues/8329))
+- Updated filter lists to increase the amount of engines that can be enabled on low memory devices. ([#8292](https://github.com/brave/brave-ios/issues/8292))
+- Updated the “Report a Broken Site” submission form to include details and contact fields. ([#8270](https://github.com/brave/brave-ios/issues/8270))
+- Updated user agent for iOS 17. ([#8155](https://github.com/brave/brave-ios/issues/8155))
+- Updated browser colors. ([#8078](https://github.com/brave/brave-ios/issues/8078))
+- Updated Search Settings page to remove “Edit” option when custom search engines are not added. ([#5987](https://github.com/brave/brave-ios/issues/5987))
+- Fixed toolbar being displayed in landscape mode when changing orientation. ([#8421](https://github.com/brave/brave-ios/issues/8421))
+- Fixed external links not working on New Tab Page. ([#8383](https://github.com/brave/brave-ios/issues/8383))
+- Fixed content blocking to be removed after disabling Ad-blocking. ([#8369](https://github.com/brave/brave-ios/issues/8369))
+- Fixed issue with pages being rendering poorly on https://youtube.com. ([#8364](https://github.com/brave/brave-ios/issues/8364))
+- Fixed tabs being cleared after relaunching browser. ([#8362](https://github.com/brave/brave-ios/issues/8362))
+- Fixed VPN status to be updated in the menu without having to open the settings page. ([#8317](https://github.com/brave/brave-ios/issues/8317))
+- Fixed context menu item when using long press action on Favorites. ([#8286](https://github.com/brave/brave-ios/issues/8286))
+- Fixed duplicated items showing when reordering Favorites. ([#8255](https://github.com/brave/brave-ios/issues/8255))
+- Fixed Favorites from showing in Private Mode on New Tab Page when using app switcher. ([#8109](https://github.com/brave/brave-ios/issues/8095))
+- Fixed favicons being rendered incorrectly for back-forward lists. ([#8206](https://github.com/brave/brave-ios/issues/8206))
+- Fixed “Learn more” link from overlapping other text in Brave Rewards panel. ([#8243](https://github.com/brave/brave-ios/issues/8243))
+- Fixed Brave News opt-in card to show after upgrading Brave. ([#8167](https://github.com/brave/brave-ios/issues/8167))
+- Fixed crash when opening Brave Rewards panel for the first time. ([#8109](https://github.com/brave/brave-ios/issues/8109))
+- Fixed crash when reordering tabs in the tab bar. ([#8074](https://github.com/brave/brave-ios/issues/8074))
+- Fixed opening Private tab using Brave app icon for certain cases. ([#8017](https://github.com/brave/brave-ios/issues/8017))
+- Fixed issue with bottom toolbar when “Tabs Bar” is set to “Landscape.” ([#8009](https://github.com/brave/brave-ios/issues/8009))
+- Fixed hamburger menu to open when using windowed mode on iPads. ([#8004](https://github.com/brave/brave-ios/issues/8004))
```

Comment:
Lets phrase it re: what was fixed rather than what it should be doing. I guess it's the same but at least this sounds better re: `Fixed etc..`.

```
- Fixed hamburger menu not being opened when using windowed mode on iPads. ([#8004](https://github.com/brave/brave-ios/issues/8004))
```